src/CloudGraphsDFG/services/CommonFunctions.jl

```diff
@@ -110,18 +112,22 @@ Will return all sessions because of 'node:Session'.
 If orderProperty is not ==, then 'order by n.{orderProperty} will be appended to the query'.
 So can make orderProperty = label or id.
 """
-function _getNeoNodesFromCyphonQuery(neo4jInstance::Neo4jInstance, matchCondition::String, orderProperty::String="")::Vector{Neo4j.Node}
+function _getNeoNodesFromCyphonQuery(neo4jInstance::Neo4jInstance, matchCondition::String, orderProperty::String=""; currentTransaction::Union{Nothing, Neo4j.Transaction}=nothing)::Vector{Neo4j.Node}
```

Comment:
Also reminder that we decided against Julia function return types in general, here its `)::Vector{Neo4j.Node}`

Reply:
Lol, I'm a fan or strong typing everything so that it's more readable - is there a Julia-related reason why?

Reply:
The abstract return types gave me some issues as it calls `convert` and then `assert` automatically. I don't know about this case though.